controllers/article.controller.js

```diff
@@ -0,0 +1,18 @@
+const { selectArticleById } = require('../models/article.model');
+
+
+exports.getArticleById = (req, res, next) => {
+	const { article_id } = req.params;
+
+	if (!Number(article_id)) {
+		return res.status(400).json({ message: 'Invalid article_id' });
+	}
```

Comment:
You can remove this logic and allow the _article_id_ to go into the PSQL query, if it is not a number it will throw a PSQL error which you can then handle yourself